lib/karto_sdk/m3rsm_test/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+include_directories (
+  ${PROJECT_SOURCE_DIR}/m3rsm_test/gtest/include
+  ${PROJECT_SOURCE_DIR}/m3rsm_test/gtest
+  ${PROJECT_SOURCE_DIR}/m3rsm_test
+  ${PROJECT_BINARY_DIR}
+)
+
+# configure_file (test_config.h.in ${PROJECT_BINARY_DIR}/include/rndf_gazebo_plugin/test_config.h)
+
+# Build gtest
+add_library(gtest STATIC gtest/src/gtest-all.cc)
```

Comment:
@LucasJSch FYI: you don't need all this, `ament_add_gtest` will take care of everything if used like it is used [here](https://github.com/ekumenlabs/slam_toolbox/blob/3136f6e45200f6821b7c69d68402d6cc93638e54/CMakeLists.txt#L173-L180).